kindtest.sh

```diff
@@ -36,7 +36,7 @@ script="${BASH_SOURCE[0]}"
 scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
 
 function usage {
-  echo "usage: ${script} [-v <version>] [-n <name>] [-o <directory>] [-t <tests>] [-c <name>] [-p true|false] [-h]"
+  echo "usage: ${script} [-v <version>] [-n <name>] [-o <directory>] [-t <tests>] [-c <name>] [-p true|false] [-th <number_of_threads>] [-h]"
```

Comment:
Generally the command line arguments are single letter. we can make add -p <parallel thread count>

Reply:
-p is already there, -t and -n are also there, do you have any suggestions?

Reply:
I am also out of letters use some letter like -x